src/ripple/rpc/handlers/AccountTx.cpp

```diff
@@ -389,6 +389,20 @@ doAccountTxJson(RPC::JsonContext& context)
     AccountTxArgs args;
     Json::Value response;
 
+    // The document states that binary and forward params is a boolean value,
```

Comment:
what do you mean by "document" here? Is it the XRPL documentation website? Can you include a link for the same?